### DISCUSSION TITLE
Can I convert and use my existing Java13 project as native-images?

### DISCUSSION THREAD

**Discussion by GuptaPratik** [oracle/graal#3234] [Q&A] [answered]
I have an existing spring boot project in which I am using Java13. So, can I convert it into a native image? and run it as a service doing "./<service-Name>".

**Comment by Thihup** [comment]
This may help https://blog.frankel.ch/kick-spring-native-tires/

**Comment by petermz** [comment, marked as answer]
Spring-native also has a step by step guide: https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/
